Gemfile

```diff
@@ -234,6 +234,9 @@ group :production do # we don"t install these on travis to speed up test runs
   gem "asset_sync", "1.1.0", require: false
 end
 
+# Newrelic support
+gem 'newrelic_rpm'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.